doc/docker/jenkinsnode/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:stretch
+
+ARG SWIGVERSION="3.0.11"
+
+# install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install \
+  apt-utils \
+  curl \
+  build-essential \
+  clang \
+  autotools-dev \
+  automake \
+  cmake \
+  pkg-config \
+  doxygen \
+  graphviz \
+  bison \
+  ruby-dev \
+  python-dev \
+  python3-dev \
+  libpython3-dev \
+  liblua5.3-dev \
+  tclcl-dev \
+  libaugeas-dev \
+  libyajl-dev \
+  libgit2-dev \
+  libboost-all-dev \
+  libssl-dev \
+  libdbus-1-dev \
+  libpcre3-dev \
+  libpcre++-dev \
+  libglib2.0-dev \
+  swig \
+  libyaml-cpp-dev \
+  libuv1-dev \
+  libxerces-c-dev \
+  checkinstall \
```

Comment:
Still needed? (It was a hack to build debian packages in an earlier docker file)

Reply:
-- Exclude Plugin xerces because XercesC library libxerces-c-dev not found
at least this is definately needed.
-- Exclude Plugin yamlcpp because yaml-cpp (libyaml-cpp-dev >= 0.5) not found
and that as well